### ISSUE TITLE
Hi, Some links have failed and some figures miss.

### ISSUE THREAD

**Issue by iScottMark** [teaghan/Cycle_SN#4]
In your [technical write-up](https://github.com/teaghan/Cycle_SN/blob/master/docs/README.md), there are some links (e.g.  cross-validation set) and some figures (e.g. Fig.4) miss or fail. Please check again and improve them. Thank you!

**Comment by teaghan** [comment]
Thanks @iScottMark, the technical write-up is a bit out-dated at this point, but I will update this ASAP.

**Comment by iScottMark** [comment]
Thank you for your great works! At present, I'm studying the details of your paper. I have learnt some original ideas and some training tricks aimed to spectra instead of images or texts in the field of TRADITIONAL DL. However, as for GAN, I wonder in the future,  whether you take the transformations of GAN(like WGAN, DCGAN) into account.

**Comment by teaghan** [comment]
Hi @iScottMark sorry for the delay; we have been working on finalizing the edits for submission. I haven't gotten around to editing the technical write-up, but perhaps the current version of our [paper](https://arxiv.org/abs/2007.03109) will serve as a useful guide. Please let me know if you have any questions. As for applying new GAN techniques, this is definitely of interest to this project, but we haven't had the time to try them.

**Comment by iScottMark** [comment]
@teaghan Hi! I have found some differences between 2 versions. (Formula, Imgs,...) But, I wonder  a calibrated line list by Cargile et al. (in prep.)

**Comment by iScottMark** [comment]
@teaghan Hi! I have found some differences between 2 versions. (Formula, Imgs,...) But, I wonder when _a calibrated line list by Cargile et al. (in prep.)_ mentioned in your paper & Ting's can be provided?

**Comment by teaghan** [comment]
Hi @iScottMark 
Unfortunately, the paper is still under construction. I am not sure when it will be available as I am not associated with the project. Hopefully we can see it soon though!
